### PR TITLE
Removes mention of @ResponseBody for the greeting method.

### LINKED ISSUE #30: greeting method in GreetingController is missing @ResponseBody annotation.
The guide mentions `@ResponseBody` (`README.adoc`, line 219) but the annotation is both missing in the `GreetingController` listing and the accompanying example code.

I'd be happy to make a PR :).

Thank you.

### PR DESCRIPTION
**Pull request by Nico769**
Fixes #30 as discussed.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Nico769 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/gs-rest-service-cors&pullRequestId=31)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-guides/gs-rest-service-cors&pullRequestId=31) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by Nico769** [comment]
@pivotal-issuemaster This is an Obvious Fix

**Comment by pivotal-issuemaster** [comment]
@Nico769 This Pull Request contains an obvious fix. Signing the Contributor License Agreement is not necessary.

**Comment by dsyer** [comment]
That's great thanks! How about also mentioning that `@RestController` is causing the result to be rendered as the response body? We could even use the same wording as before.

File: README.adoc [hunk 1]
```diff
@@ -216,14 +216,12 @@ relying on a view technology to perform server-side rendering of the greeting da
...
-support, you need not do this conversion manually. Because {jackson}[Jackson] is on the
-classpath, Spring's
+To accomplish this, the {RestController}[`@RestController`] annotation
+assumes that every method inherits the {ResponseBody}[`@ResponseBody`] semantics by default.
+Therefore, a returned object data is inserted directly into the response body.
+
+Thanks to Spring's HTTP message converter support, the `Greeting` object is naturally converted to JSON.
+Because {jackson}[Jackson] is on the classpath, Spring's
 {MappingJackson2HttpMessageConverter}[`MappingJackson2HttpMessageConverter`] is
 automatically chosen to convert the `Greeting` instance to JSON.
```
Comment (Nico769) [inline README.adoc:226]: > That's great thanks! How about also mentioning that `@RestController` is causing the result to be rendered as the response body? We could even use the same wording as before.

How about this?

Comment (dsyer) [reply README.adoc:226]: LGTM

**Comment by Nico769** [comment]
As a side note, if it is not too much trouble, I'd appreciate if whoever merges this could label the PR as `hacktoberfest-accepted`since I'm participating to Hacktoberfest this year.

Many thanks! :)

**Pull request merged**